test/e2e/app-dir/ecmascript-features/app/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+import { abc } from './export-as'
+import json from './file.json' with { type: 'json' }
+// Test both server and client compilation of ECMAScript features.
+
+class ClassWithPrivate {
```

Comment:
Specifically this is missing `#privateField in this`. Which shipped distinctly from private fields in general, as "ergonomic brand checks"

Reply:
Thanks! Added in https://github.com/vercel/next.js/pull/65613/files#diff-b8678af04064ee7c1e38b7412725f6188ee4a4e6771ad7473d40db33cf40e378R37. That one does not work with webpack indeed, left a todo there.